libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -127,6 +132,17 @@ SymbolicFunctionVariable::SymbolicFunctionVariable(
 	solAssert(m_type->category() == solidity::Type::Category::Function, "");
 }
 
+SymbolicFunctionVariable::SymbolicFunctionVariable(
```

Comment:
I am a bit unhappy with the code repitition seen here, but I can't think of anything better either

Reply:
Do you mean between the constructors of `SymbolicFunctionVariable` or also regarding the other classes?

Reply:
I was mainly referring to the c'tors

Reply:
Right,yeah, it's hard to reuse one from the other. We could have a simpler 3rd constructor with the common parameters that is called by the others, but this would only increase code...